src/option/index.ts

```diff
@@ -1 +1,32 @@
-export default {}
+import { assertIsHexStr } from '../utils'
+import { deserializeArray } from '../array'
+
+export const serializeOption = (origin: TypeElement[]) => {
```

Comment:
Why is the type of `option`'s argument `[string, any]`?

Reply:
No. `declare type TypeElement = [string, string]` not `[string, any]`

Reply:
According to the doc, the `option` has zero or one element, so what does the first string do here?

Reply:
Updated. https://github.com/Keith-CY/molecule-javascript/pull/12/commits/2bfe30878dc633b5ad6c65dfb5d74d425c3b6604